### PR TITLE
Setting up UI on Quartz

### PR DESCRIPTION
**Pull request by JonathanXu1**
Used React-Bootstrap to set up card layout and navigation elements for the Electron app. Customized window bar. The design was based off Paul's proposal:
https://files.slack.com/files-pri/TL5DQ6EQY-FNHAJ5UTS/desktop.png

### CODE REVIEW

File: src/client/App.jsx [hunk 1]
```diff
@@ -8,14 +9,23 @@ import { AuthConsumer } from './layers/Auth.jsx';
...
 
+class App extends React.Component {
     render() {
        return (
          <div>
+             <TitleBar
+             theme={{
+               barHeight: "48px",
+               winBarHeight: "48px",
+               barBackgroundColor: "#292E41"
```
Comment (hanburger97) [inline src/client/App.jsx]: In the future let's not hard code the theme 🙄, you can define them in .less files and import them here

File: src/client/layers/Auth.jsx [hunk 2]
```diff
@@ -7,25 +7,25 @@ import config from '../utils/config.js';
...
-  
+
   constructor(props) {
     super(props);
     this.updateAuth = this.updateAuth.bind(this);
     this.login = this.login.bind(this);
     this.state = {
         miner: null,
-        authenticated: false,
+        authenticated: true, //TODO: Change back
```
Comment (hanburger97) [inline src/client/layers/Auth.jsx]: hmm this shouldn't be committed

Comment (JonathanXu1) [reply src/client/layers/Auth.jsx]: Sorry, my bad. I'll undo that

File: src/client/App.jsx (hunk 1 above)
Comment (JonathanXu1) [reply src/client/App.jsx]: I followed the example to change the theme object here: 
https://github.com/Cristian006/frameless-titlebar/blob/master/src/TitleBar/Theme/index.js
Is there a way to use .less files create javascript objects?

Comment (hanburger97) [reply src/client/App.jsx]: Thats a good question, Front end dev is opinionated and theres a lot of approach/school of thoughts, you can looked into styled components I guess. But I wont block this PR for now 😄

**Pull request merged**